DNN Platform/Library/Services/UserProfile/UserProfilePicHandler.cs

```diff
@@ -163,16 +163,7 @@ public void ProcessRequest(HttpContext context)
             context.Response.Cache.SetExpires(DateTime.Now.AddMinutes(1));
             context.Response.Cache.SetMaxAge(new TimeSpan(0, 1, 0));
             context.Response.AddHeader("Last-Modified", DateTime.Now.ToString("r"));
-
-	        try
-	        {
-				context.Response.End();
-	        }
-			catch (ThreadAbortException)//if ThreadAbortException will shown, should catch it and do nothing.
-			{
-
-			}
-            
+            context.ApplicationInstance.CompleteRequest();
```

Comment:
Oliver is correct.  We should not be using response.end() as it is evil.  Instead we should be doing CompleteRequest() as it is more friendly and won't drag down server performance or fill the eventlog with useless error messages.
(http://stackoverflow.com/questions/1087777/is-response-end-considered-harmful)